### PR TITLE
chore(examples): add getting started example on all flavors

### PR DESCRIPTION
**Pull request by dhayab**
This PR adds the same "Getting started" example for all flavors of the monorepo. Most of the other examples are more complex, so this should allow people to iterate more easily when contributing on the library.

CodeSandboxes:
- [InstantSearch.js](https://codesandbox.io/s/github/algolia/instantsearch/tree/chore/getting-started-examples/examples/js/getting-started?file=/src/app.js)
- [React InstantSearch](https://codesandbox.io/s/github/algolia/instantsearch/tree/chore/getting-started-examples/examples/react/getting-started)
- [React InstantSearch Hooks](https://codesandbox.io/s/github/algolia/instantsearch/tree/chore/getting-started-examples/examples/react-hooks/getting-started?file=/src/App.tsx)
- [Vue InstantSearch](https://codesandbox.io/s/github/algolia/instantsearch/tree/chore/getting-started-examples/examples/vue/getting-started)

FX-2164

### CODE REVIEW

File: examples/vue/getting-started/src/main.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Vue from 'vue';
+import InstantSearch from 'vue-instantsearch';
+import App from './App.vue';
+
+Vue.config.productionTip = false;
+
+Vue.use(InstantSearch);
+
+new Vue({
+  render: h => h(App),
```
Comment (Haroenv) [inline examples/vue/getting-started/src/main.js:10]:
```suggestion
  render: (h) => h(App),
```

File: examples/js/getting-started/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Getting started demo
+
+_This project was generated with [create-instantsearch-app](https://github.com/algolia/instantsearch.js/tree/master/packages/create-instantsearch-app) by [Algolia](https://algolia.com)._
```
Comment (FabienMotte) [inline examples/js/getting-started/README.md:3]:
```suggestion
_This project was generated with [create-instantsearch-app](https://github.com/algolia/instantsearch/tree/master/packages/create-instantsearch-app) by [Algolia](https://algolia.com)._
```

File: examples/react-hooks/getting-started/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# react-instantsearch-hooks-app
+
+_This project was generated with [create-instantsearch-app](https://github.com/algolia/instantsearch.js/tree/master/packages/create-instantsearch-app) by [Algolia](https://algolia.com)._
```
Comment (FabienMotte) [inline examples/react-hooks/getting-started/README.md:3]:
```suggestion
_This project was generated with [create-instantsearch-app](https://github.com/algolia/instantsearch/tree/master/packages/create-instantsearch-app) by [Algolia](https://algolia.com)._
```

File: examples/react/getting-started/README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# react-instantsearch-app
+
+_This project was generated with [create-instantsearch-app](https://github.com/algolia/instantsearch.js/tree/master/packages/create-instantsearch-app) by [Algolia](https://algolia.com)._
```
Comment (FabienMotte) [inline examples/react/getting-started/README.md:3]:
```suggestion
_This project was generated with [create-instantsearch-app](https://github.com/algolia/instantsearch/tree/master/packages/create-instantsearch-app) by [Algolia](https://algolia.com)._
```

File: examples/vue/getting-started/README.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# vue-instantsearch-app
+
+_This project was generated with [create-instantsearch-app](https://github.com/algolia/instantsearch.js/tree/master/packages/create-instantsearch-app) by [Algolia](https://algolia.com)._
```
Comment (FabienMotte) [inline examples/vue/getting-started/README.md:3]:
```suggestion
_This project was generated with [create-instantsearch-app](https://github.com/algolia/instantsearch/tree/master/packages/create-instantsearch-app) by [Algolia](https://algolia.com)._
```

File: examples/js/getting-started/README.md (hunk 2 above)
Comment (FabienMotte) [reply examples/js/getting-started/README.md:3]: I guess we need to update all the README.md from templates in `create-instantsearch-app.
I'll take care of that.

**Review by FabienMotte — CHANGES_REQUESTED** [review]

**Review by Haroenv — APPROVED** [review]

**Pull request merged**